test/e2e/tests/metrics/attribution/mock-page/index.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Mock E2E Cookie Handler</title>
+</head>
+<script>
+	function fireCookieIdEvent() {
+    console.log('postmessage');
+		window.postMessage(
+      {
+        target: 'metamask-contentscript',
+        data: {
+          name: 'metamask-cookie-handler',
+          data: {
+            jsonrpc: '2.0',
+            method: 'getCookieFromMarketingPage',
+            params: [{ ga_client_id: 12345, origin: 'http://127.0.0.1:8080' }],
+            id: Date.now(),
+            origin: 'http://127.0.0.1:8080',
+          },
+        },
+      },
+      'http://127.0.0.1:8080',
+    );
+	}
+	window.onload = fireCookieIdEvent;
+  window.addEventListener('DOMContentLoaded', function() {
+    setTimeout(fireCookieIdEvent, 2000);
```

Comment:
Can we reduce to 1s to reduce test suit execution time?

Reply:
This was also added to reduce flakiness.